<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>712</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>670</width>
    <height>712</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>670</width>
    <height>712</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Body Damage Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>381</width>
      <height>401</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>dudeVinci.png</pixmap>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>440</y>
      <width>16</width>
      <height>61</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>440</y>
      <width>16</width>
      <height>61</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>20</width>
      <height>91</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>360</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>397</x>
      <y>340</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>150</y>
      <width>118</width>
      <height>21</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_9">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="nutHead">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="nutRhand">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="nutRleg">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>360</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="nutLleg">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>340</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="nutLhand">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>150</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="nutLbody">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="nutUbody">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="nutRfeet">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>490</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="nutLfeet">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>490</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtReport">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>550</y>
      <width>381</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkArrow">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>520</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Took an Arrow in the Knee</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="pgbHP">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="Line" name="line_10">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>20</width>
      <height>121</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_11">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>20</width>
      <height>121</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_12">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_13">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="nutRarm">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="nutLarm">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdHeal">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>95</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>HEAL</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>txtReport</zorder>
   <zorder>chkArrow</zorder>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
   <zorder>line_3</zorder>
   <zorder>line_4</zorder>
   <zorder>line_5</zorder>
   <zorder>line_6</zorder>
   <zorder>line_7</zorder>
   <zorder>line_8</zorder>
   <zorder>line_9</zorder>
   <zorder>nutLhand</zorder>
   <zorder>nutRhand</zorder>
   <zorder>nutHead</zorder>
   <zorder>nutLleg</zorder>
   <zorder>nutLfeet</zorder>
   <zorder>nutRfeet</zorder>
   <zorder>nutUbody</zorder>
   <zorder>nutLbody</zorder>
   <zorder>nutRleg</zorder>
   <zorder>pgbHP</zorder>
   <zorder>line_10</zorder>
   <zorder>line_11</zorder>
   <zorder>line_12</zorder>
   <zorder>line_13</zorder>
   <zorder>nutRarm</zorder>
   <zorder>nutLarm</zorder>
   <zorder>cmdHeal</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>670</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>nutHead</tabstop>
  <tabstop>nutRarm</tabstop>
  <tabstop>nutLarm</tabstop>
  <tabstop>nutRhand</tabstop>
  <tabstop>nutLhand</tabstop>
  <tabstop>nutUbody</tabstop>
  <tabstop>nutLbody</tabstop>
  <tabstop>nutRleg</tabstop>
  <tabstop>nutLleg</tabstop>
  <tabstop>nutRfeet</tabstop>
  <tabstop>nutLfeet</tabstop>
  <tabstop>chkArrow</tabstop>
  <tabstop>txtReport</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>changeVal()</slot>
 </slots>
</ui>
